### PR TITLE
feat: add example and explanation using crate keyword

### LINKED ISSUE #942: feat: add `crate` keyword
In the Packages and Crates section, current examples use the crate's name to traverse the module tree.

- Rework the examples to use the `crate` keyword
- Rework the explanation to introduce that keyword

### PR DESCRIPTION
**Pull request by NueloSE**
Ready for review.

each PR is made for each sub section from 7.1 to 7.5. 

This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/952)

Closes #942

### CODE REVIEW

File: src/ch07-02-defining-modules-to-control-scope.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Defining Modules to Control Scope
 
 In this section, we’ll talk about modules and other parts of the module system,
-namely _paths_ that allow you to name items and the `use` keyword that brings a
-path into scope.
+namely _paths_ that allow you to name items, the `crate` and the `use` keyword that brings a
```
Comment (enitrat) [inline src/ch07-02-defining-modules-to-control-scope.md:4]: the `crate` keyword is will actually be explained in the next section, so this line can be reverted

**Comment by enitrat** [comment]
@NueloSE can you perform a self-review first?

**Comment by NueloSE** [comment]
> @NueloSE can you perform a self-review first?

on it. 
I will notify you when am done

**Comment by NueloSE** [comment]
LGTM

> @NueloSE can you perform a self-review first?

File: src/ch07-01-packages-and-crates.md [hunk 2]
```diff
@@ -55,7 +55,11 @@ edition = "2024_07"
 # foo = { path = "vendor/foo" }
 ```
 
-As you develop your package, you may want to organize your code into multiple Cairo source files. You can do this by creating additional _.cairo_ files within the _src_ directory or its subdirectories.
+As you develop your package, you may want to organize your code into multiple Cairo source files. You can do this by creating additional _.cairo_ files within the _src_ directory or its subdirectories. When doing so, you can use the `crate` keyword to refer to the root of your crate, which helps to keep your module references clear and concise.
+
+For instance, if you have a module `foo` in `src/foo.cairo` and you want to use a function `my_function()` from that module in `lib.cairo`, you would write:
+
+```cairo,noplayground
```
Comment (enitrat) [inline src/ch07-01-packages-and-crates.md:62]:
```suggestion
As you develop your package, you may want to organize your code into multiple Cairo source files. You can do this by creating additional _.cairo_ files within the _src_ directory or its subdirectories.
```

**Review by enitrat — APPROVED** [review]

**Pull request merged**